### PR TITLE
Added functions to create commit and pr.

### PR DESCRIPTION
**Pull request by bridgetweston**

### CODE REVIEW

**Comment by bridgetweston** [comment]
Some of my functions aren't completed, but I wanted to commit anyway since I haven't in a while.

File: competencySyncScript [hunk 1]
```diff
@@ -28,11 +35,12 @@ function testRequests() {
       var fileName = file.getName(); //get the file name
```
Comment (codeallthethingz) [inline competencySyncScript]: Spacing seems a little off here

File: competencySyncScript [hunk 2]
```diff
@@ -28,11 +35,12 @@ function testRequests() {
...
       if (fileName != 'TEMPLATE') { //ignore files with name 'TEMPLATES'
         var indexOfNameInGithubFileNames = githubFileNames.indexOf(fileName);
-        
         //Checks if Drive file exists in GitHub repo (by file name)
         if (indexOfNameInGithubFileNames != -1) { //indexOf() returns -1 if value does not exist in array
```
Comment (codeallthethingz) [inline competencySyncScript]: I would combine line 37 and the insides of the if conditional as a function so you can get rid of the comment
```
if (checkIfDriveFileExists(githubFilesNames, filename)) {
}
```

File: competencySyncScript [hunk 3]
```diff
@@ -45,6 +53,116 @@ function testRequests() {
...
+  var folders = ["testFolder"];
+  var folder = DriveApp.getFoldersByName(folders[0]);
+  
+  while (folder.hasNext()) {
+    var files = folder.next().getFiles();
+    while (files.hasNext()) { //for each file in the folder
+      var file = files.next();
+      var fileData = {
+        "fileName" : file.getName()
+        //"fileDecodedContent" : file.getAs(contentType) //HOW DO I GET AS A STRING???????
```
Comment (codeallthethingz) [inline competencySyncScript]: create a function called `getFileAsString()` and put all your logic in there (and possbly a test for it too)

File: competencySyncScript [hunk 4]
```diff
@@ -45,6 +53,116 @@ function testRequests() {
...
+  }
+  
+  var content = {
+    "message" : commitMessage,
+    "content" : fileContent,
+    "sha" : sha,
+    "branch" : branchName
+  };
+  
+  var options = {
```
Comment (codeallthethingz) [inline competencySyncScript]: You can probably extract this object out into a function called `createOptions` that you pass in the content to.

File: competencySyncScript (hunk 3 above)
Comment (bridgetweston) [reply competencySyncScript]: Done.

File: competencySyncScript (hunk 4 above)
Comment (bridgetweston) [reply competencySyncScript]: Done.

**Pull request closed without merging**